bundle/manifests/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -439,7 +439,7 @@ spec:
                 - --upstream=http://127.0.0.1:8383/
                 - --logtostderr=true
                 - --v=0
-                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
```

Comment:
oh `v0.8.0` :rabbit:

Reply:
Yes. It seems we missed upgrading this at some point